plugins/src/core/ddl.d.ts

```diff
@@ -0,0 +1,10 @@
+interface DeepLinkContent {
+    title?: string;
+    description?: string;
+}
+export interface DeepLink {
+    content: DeepLinkContent;
+    data: any;
+    url: string;
+}
+export {};
```

Comment:
?

Reply:
this file  was created automatically after the bundling, I'm not sure why the export {}; is needed, seems weird, I'll delete that.